src/adaPets.js

```diff
@@ -1,43 +1,90 @@
-// Use Node-style imports for dependencies.
-const axios = require('axios');
+// Node-style imports for dependencies
+const axios  = require('axios');
 const result = require('./result.js');
 
 const setResult = result.setResult;
-const setError = result.setError;
+const setError  = result.setError;
 
 const BASE_URL = 'http://localhost:3000/pets/';
 
-// Option functions.
+// Option functions
 const listPets = () => {
-  // Fill out as part of Wave 1.
+
+  axios.get(BASE_URL)
+
+  .then((response) => {
+    allPets = response.data.map((petData) => (
+      { id: petData.id, name: petData.name } // mapping just the required fields
+    ));
```

Comment:
You forgot to use `const` here.  (This will create a global variable.)

```suggestion
    const allPets = response.data.map((petData) => (
      { id: petData.id, name: petData.name } // mapping just the required fields
    ));
```